cmd/podman/containers/restore.go

```diff
@@ -137,6 +165,9 @@ func restore(cmd *cobra.Command, args []string) error {
 	if argLen < 1 && !restoreOptions.All && !restoreOptions.Latest && restoreOptions.Import == "" {
 		return errors.Errorf("you must provide at least one name or id")
 	}
+	if argLen > 1 && restoreOptions.Name != "" {
+		return errors.Errorf("--name can only be used with one checkpoint image")
```

Comment:
Minor nit (non-blocking): consider using `fmt.Errorf` and `errors.New` instead of the non-standard `errors` package.